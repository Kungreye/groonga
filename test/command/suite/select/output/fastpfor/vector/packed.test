table_create Entries TABLE_PAT_KEY ShortText
column_create Entries contents COLUMN_VECTOR|COMPRESS_FASTPFOR Text

load --table Entries
[
  {
    "_key": "Groonga",
    "contents": [
      "I found Groonga.",
      "It's a fast fulltext search engine!"
    ]
  },
  {
    "_key": "Mroonga",
    "contents": [
      "I found Mroonga.",
      "It's a MySQL storage engine to use Groonga!"
    ]
  }
]

select Entries
