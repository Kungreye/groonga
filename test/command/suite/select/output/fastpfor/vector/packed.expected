table_create Entries TABLE_PAT_KEY ShortText
[[0,0.0,0.0],true]
column_create Entries contents COLUMN_VECTOR|COMPRESS_FASTPFOR Text
[[0,0.0,0.0],true]
load --table Entries
[
  {
    "_key": "Groonga",
    "contents": [
      "I found Groonga.",
      "It's a fast fulltext search engine!"
    ]
  },
  {
    "_key": "Mroonga",
    "contents": [
      "I found Mroonga.",
      "It's a MySQL storage engine to use Groonga!"
    ]
  }
]
[[0,0.0,0.0],2]
select Entries
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        2
      ],
      [
        [
          "_id",
          "UInt32"
        ],
        [
          "_key",
          "ShortText"
        ],
        [
          "contents",
          "Text"
        ]
      ],
      [
        1,
        "Groonga",
        [
          "I found Groonga.",
          "It's a fast fulltext search engine!"
        ]
      ],
      [
        2,
        "Mroonga",
        [
          "I found Mroonga.",
          "It's a MySQL storage engine to use Groonga!"
        ]
      ]
    ]
  ]
]
